### PR TITLE
support int32 and int64 kernel for clip operator

### PR DESCRIPTION
**Pull request by wuyefeilin**
### PR types
Function optimization 

### PR changes
OPs

### Describe
clip支持int32和int64 kernel

### CODE REVIEW

**Review by Avin0323 — DISMISSED** [review]
LGTM for OP-benchmark-CI

**Review by TCChenlong — DISMISSED** [review]
LGTM for API Doc
TODO：update Chinese API Doc

File: python/paddle/tensor/math.py [hunk 1]
```diff
@@ -1475,10 +1475,10 @@ def clip(x, min=None, max=None, name=None):
...
 
     Args:
-        x (Tensor): An N-D Tensor with data type float32 or float64.
-        min (float32|Tensor): The lower bound with type ``float32`` or a ``Tensor``
+        x (Tensor): An N-D Tensor with data type float32, float64, int32 or int64.
+        min (float32|int|Tensor): The lower bound with type ``float32`` , ``int`` or a ``Tensor``
             with shape [1] and type ``int32``, ``float32``, ``float64``.
-        max (float32|Tensor): The upper bound with type ``float32`` or a ``Tensor``
+        max (float32|int|Tensor): The upper bound with type ``float32``, ``int`` or a ``Tensor``
             with shape [1] and type ``int32``, ``float32``, ``float64``.
```
Comment (wawltor) [inline python/paddle/tensor/math.py:1482]: float32 -> float

File: python/paddle/tensor/math.py [hunk 2]
```diff
@@ -1503,16 +1503,25 @@ def clip(x, min=None, max=None, name=None):
...
     if in_dygraph_mode():
         if isinstance(min, Variable):
             min = min.numpy().item(0)
         if isinstance(max, Variable):
             max = max.numpy().item(0)
-        min = fmin if min is None else min
-        max = fmax if max is None else max
+        min = min_ if min is None else min
+        max = max_ if max is None else max
+        print('input min max value: ', min, max)
```
Comment (wawltor) [inline python/paddle/tensor/math.py]: 删除print

**Review by wawltor — DISMISSED** [review]

**Review by wawltor — APPROVED** [review]
LGTM

**Review by TCChenlong — APPROVED** [review]
LGTM

**Pull request merged**